### PR TITLE
namespace fix

### PR DESCRIPTION
**Pull request by someburner**
The reason for the fields error is veevalidate and vuetable-2 both make use of `fields`, which for whatever reason is not scoped correctly.  

Not totally sure this will compile cause my local project has diverged significantly.

### CODE REVIEW

**Comment by someburner** [comment]
And cause I was curious about the root cause- I found this in a vee validate issue:

> Unfortunately custom components cannot set its scope from the parent form tag, this is only available for the native HTML inputs since they have a native form attribute which can be inferred to obtain the form tag.

IMO projects should make disclaimers for these then.

**Comment by alexanderrudnik** [comment]
@someburner  I appreciate your help. Thanks a lot!

File: src/components/forms/form-elements/FormElements.vue [hunk 1]
```diff
@@ -285,8 +285,9 @@
     computed: {
       isSuccessfulEmailValid () {
         let isValid = false
-        if (this.fields.successfulEmail) {
-          isValid = this.fields.successfulEmail.validated && this.fields.successfulEmail.valid
+        if (
```
Comment (alexanderrudnik) [inline src/components/forms/form-elements/FormElements.vue]: wrong if condition :)

Comment (someburner) [reply src/components/forms/form-elements/FormElements.vue]: Just keeping you on your toes ;)

**Pull request merged**